tests/dump_input/port/appl_db.json

```diff
@@ -0,0 +1,35 @@
+{	
```

Comment:
Why do we need to add new mock db tables here? 
Can tests/mock_tables/*.json be used?

Reply:
There might be multiple modules coming in as a part of this infra. So using the same json file source would get increasingly cumbersome & big to handle. 

Also, think of this case where multiple dump modules and other tests might have to look into the same tables. As the files become large, the implementers would have to deal with any dependencies involved between tables and need to make sure other tests aren't failing. So, to decouple and keep the implementation clean and easy, decided to separate out data sources into different JSON files